examples/animation/locomotion.html

```diff
@@ -0,0 +1,433 @@
+        });
+        cameraEntity.translateLocal(0, 5, 15);
+        cameraEntity.rotateLocal(-20, 0, 0);
+        app.root.addChild(cameraEntity);
+
+        app.scene.ambientLight = new pc.Color(0.5, 0.5, 0.5);
+        // Create an entity with a light component
+        var light = new pc.Entity();
+        light.addComponent("light", {
+            type: "directional",
+            color: new pc.Color(1, 1, 1),
+            castShadows: true,
+            intensity: 1,
+            shadowBias: 0.2,
+            shadowDistance: 5,
+            normalOffsetBias: 0.05,
+            shadowResolution: 2048
+        });
+        light.setLocalEulerAngles(45, 30, 0);
+        app.root.addChild(light);
+
+        // Load model and animation assets
+        var model = null;
+        var idleAnim = null;
+        var walkAnim = null;
+        var strafeLeftAnim = null;
+        var strafeRightAnim = null;
+        var jogAnim = null;
+
+        var stateGraphAsset = new pc.Asset('stateGraph', 'animstategraph', null, animStateGraph);
+        stateGraphAsset.once('load', function (asset) {
+            var stateGraph = asset.resource;
+        app.assets.loadFromUrl("../assets/models/bitmoji.glb", "container", function (err, asset) {
+            model = asset;
+        app.assets.loadFromUrl("../assets/animations/bitmoji/idle.glb", "container", function (err, asset) {
+            idleAnim = asset;
+        app.assets.loadFromUrl("../assets/animations/bitmoji/walk.glb", "container", function (err, asset) {
+            walkAnim = asset;
+        app.assets.loadFromUrl("../assets/animations/bitmoji/run.glb", "container", function (err, asset) {
+            jogAnim = asset;
+        app.assets.loadFromUrl("../assets/animations/bitmoji/jump-flip.glb", "container", function (err, asset) {
+            jumpAnim = asset;
+
+            app.start();
+
+            var modelEntity = new pc.Entity();
+            modelEntity.name = 'modelEntity';
+
+            modelEntity.addComponent("model", {
+                enabled: true,
+                type: "asset",
+                asset: model.resource.model,
+                castShadows: true
+            });
+
+            modelEntity.addComponent('anim', {
+                activate: true,
+            });
+
+            modelEntity.anim.loadStateGraph(stateGraph);
+            var locomotionLayer = modelEntity.anim.baseLayer;
+            locomotionLayer.assignAnimation('Idle', idleAnim.resource.animations[0].resource);
+            locomotionLayer.assignAnimation('Walk', walkAnim.resource.animations[0].resource);
+            locomotionLayer.assignAnimation('Jog', jogAnim.resource.animations[0].resource);
+            locomotionLayer.assignAnimation('Jump', jumpAnim.resource.animations[0].resource);
+
+            app.root.addChild(modelEntity);
+
+            var planeEntity = new pc.Entity();
+            planeEntity.name = 'Plane';
+            planeEntity.addComponent("model", {
```

Comment:
"render"

Reply:
actually, this might not work if it's collider.

Reply:
actually yes, it should be a render.
the collider is not mesh collider, so it does not use the model